larq_compute_engine/tflite/kernels/bconv2d_impl.h

```diff
@@ -25,14 +25,13 @@ inline void im2col(const ConvParams& params, const RuntimeShape& input_shape,
                    const RuntimeShape& filter_shape,
                    const RuntimeShape& output_shape,
                    const RuntimeShape& im2col_shape, TBitpacked* im2col_data,
-                   RuntimeShape& result_shape, const TBitpacked** result_data,
-                   const std::int32_t zero_point) {
+                   RuntimeShape& result_shape, const TBitpacked** result_data) {
   const int stride_width = params.stride_width;
   const int stride_height = params.stride_height;
   const int dilation_width_factor = params.dilation_width_factor;
   const int dilation_height_factor = params.dilation_height_factor;
 
-  const std::uint8_t zero_byte = (std::uint8_t)zero_point;
+  const std::uint8_t zero_byte = 0;
```

Comment:
Do you think that for clarity/readability, it makes sense add a comment here that this `0` means it'll be `+1` padding? (Conversely, writing `~0` or `0xff` here would result in `-1` padding.) I do like that the `zero_point` argument is removed from the function arguments itself.
```suggestion
  // On bitpacked data, padding with `0` effectively pads with `+1.0` values.
  const std::uint8_t zero_byte = 0;
```

Reply:
Good point :+1: